tests/views/test_message.py

```diff
@@ -129,3 +129,33 @@ def test_empty_subject_and_body_rejected(self):
 
         self.assertIn("Please enter a subject".encode(), response.data)
         self.assertIn("Please enter a message".encode(), response.data)
+
+    message_form = {'body': "TEST BODY",
+                    'subject': "TEST SUBJECT"}
+
+    @requests_mock.mock()
+    def test_form_submit_with_valid_data(self, mock_request):
+        mock_request.post(f'{app.config["BACKSTAGE_API_URL"]}/v1/secure-message/send-message', status_code=201)
+        mock_request.get(f'{app.config["BACKSTAGE_API_URL"]}/v1/secure-message/messages', json={}, status_code=200)
+
+        with app.app_context():
+            response = self.app.post(self.create_message_url, data=self.message_form, follow_redirects=True)
+
+        self.assertIn("Message sent.".encode(), response.data)
```

Comment:
A bit picky sorry but can you do a few asserts for a few elements that appear on the inbox page.  With just this it's not immediately obvious that it's on the inbox page